.gitattributes

```
@@ -0,0 +1,42 @@
+*.pbxproj binary merge=union
+
+# Consistent line endings for text files
+*.c text
+*.cpp text
+*.h text
+*.py text eol=lf
+*.java text eol=lf
+*.js text eol=lf
+*.ts text eol=lf
+*.css text eol=lf
+*.html text eol=lf
+*.md text eol=lf
+*.sh text eol=lf
+*.xml text eol=lf
+*.json text eol=lf
+*.yml text eol=lf
+*.yaml text eol=lf
+
```

Let's add some more Python specifics:
```suggestion
*.pxd    text diff=python eol=lf
*.py     text diff=python eol=lf
*.py3    text diff=python eol=lf
*.pyw    text diff=python eol=lf
*.pyx    text diff=python eol=lf
*.pyz    text diff=python eol=lf
*.pyi    text diff=python eol=lf

*.db     binary
*.p      binary
*.pkl    binary
*.pickle binary
*.pyc    binary export-ignore
*.pyo    binary export-ignore
*.pyd    binary
```